onboarding/serializers.py

```diff
@@ -267,6 +282,19 @@ class Meta:
         )
 
 
+class PackagePagesForUsersSerializer(serializers.ModelSerializer):
```

Comment:
Interesting, so one of a fields is another serialiser?

Reply:
Relational serialization:
result of the requests includes a list of pages related by the foreign key :slightly_smiling_face: 
The keys of related list depends of its serializer.